scico/linop/_stack.py

```diff
@@ -85,7 +85,7 @@ def __init__(
     def _eval(self, x: JaxArray) -> Union[JaxArray, BlockArray]:
         if self.collapsable and self.collapse:
             return snp.stack([op @ x for op in self.ops])
-        return BlockArray.array([op @ x for op in self.ops])
+        return BlockArray([op @ x for op in self.ops])
```

Comment:
Any opinions on the syntax `snp.BlockArray(stuff)` vs `snp.blockarray(stuff)`? The second one is little more like NumPy.

Reply:
I'd lean towards copying NumPy style, but this is worth a discussion.

Reply:
I made both do the same thing, but tended to write `snp.blockarray(...)` wherever I could. By comparison, `np.ndarray` and `np.array` do different things, with `np.ndarray` documented as a low-level method.

Reply:
I think that's OK, but perhaps add a note in the docs indicating that the `snp.blockarray` form is preferred.